resilience4j-spring-boot2/build.gradle

```diff
@@ -1,6 +1,9 @@
 dependencies {
     compile project(':resilience4j-spring')
     compile project(':resilience4j-micrometer')
+    compile project(':resilience4j-reactor')
+    compile(libraries.spring_boot2_webflux)
```

Comment:
I think these should be `compileOnly`. 
There are users who doesn't use reactor or webflux.
What do you think?

Reply:
I see your point.. rxjava is already part of testCompile.  I could use Flowable instead of FLUX (thus webflux).  
rxjava is anyway part of the manifest of springboot2.  Would that be OK ?

Reply:
Flowable also cannot be used because rxjava is only used in test. 
I think SSE endpoint should be only enabled when Flux exists by `ConditionalOnClass(Flux.class)`.

Reply:
makes sense to me. will let you know once the changes are done.

Reply:
Updated it with conditiononclass

Reply:
Need to be `compileOnly`. Please change these :)

Reply:
done.  @dlsrb6342 . Thank you